dashboard/app/models/levels/poetry.rb

```diff
@@ -73,6 +73,10 @@ def self.create_from_level_builder(params, level_params)
     )
   end
 
+  def uses_google_blockly?
+    true
```

Comment:
🤞

Reply:
If you don't mind, can you wait until tomorrow or friday when the location picker bug is fixed?

Reply:
yeah, of course!